src/Mod/PartDesign/App/FeatureFillet.cpp

```diff
@@ -73,7 +76,18 @@ App::DocumentObjectExecReturn *Fillet::execute(void)
     } catch (Base::Exception& e) {
         return new App::DocumentObjectExecReturn(e.what());
     }
-    std::vector<std::string> SubNames = std::vector<std::string>(Base.getSubValues());
+    std::vector<std::string> SubNames = std::vector<std::string>(Base.getSubValues());  
+
+    if (UseAllEdges.getValue()){
+        SubNames.clear();
+        int count = TopShape.countSubElements("Edge");
+        for (int ii = 0; ii < count; ii++){
+            std::ostringstream edgeName;
+            edgeName << "Edge" << ii+1;
+            SubNames.push_back(edgeName.str());
+        }

```

Comment:
I expect this to be invalid once topo-naming is merged---edges will start having names like `Edge_Between_Face1_and_Face2`. It would be better to add loop through the edges themselves and add their names. I'll see if I can't find a snippet for this.

Reply:
Is that really how the topo naming algorithm works? That's not how I understood it.

Reply:
If this happens it will break a lot of stuff, especially addons depending on these subshape names.

I have found a way to get the type name (e.g. "Edge", "Vertex", "Face") from the sub shape object.  I will update the PR to use that instead of hard-coding "Edge".

Reply:
@adrianinsaval I won't pretend to understand if that that's how it works so I might be wrong, but I do believe running through the edges and adding their names to the list (or the edges themselves) is better than creating labels which may or may not work.

Reply:
I guess it's safer but my understanding is that we will still have EdgeN as the names for edges, there's a separate mapped name that get's the history stuff and it's a bunch of letters and numbers not long descriptive names like that, the algorithm then finds which edge number corresponds to the mapped name.

Reply:
I see. If so that's good.